projects.go

```diff
@@ -672,6 +673,7 @@ type EditProjectOptions struct {
 	PackagesEnabled                           *bool                                `url:"packages_enabled,omitempty" json:"packages_enabled,omitempty"`
 	ServiceDeskEnabled                        *bool                                `url:"service_desk_enabled,omitempty" json:"service_desk_enabled,omitempty"`
 	AutocloseReferencedIssues                 *bool                                `url:"autoclose_referenced_issues,omitempty" json:"autoclose_referenced_issues,omitempty"`
+	SuggestionCommitMessage			  *string			       `url:"suggestion_commit_message,omitempty" json:"suggestion_commit_message,omitempty"`
```

Comment:
I guess we should also add the field to the `Project` struct right? Just below the `AutocloseReferencedIssues` field to keep the ordering inline with the docs...

Reply:
Added. Thank you.